src/histolab/slide.py

```diff
@@ -441,10 +441,10 @@ def _wsi(self) -> Union[openslide.OpenSlide, openslide.ImageSlide]:
         """
         try:
             slide = openslide.open_slide(self._path)
-        except openslide.OpenSlideError:
+        except PIL.UnidentifiedImageError:
             raise openslide.OpenSlideError(
                 "Your wsi has something broken inside, a doctor is needed"
```

Comment:
We're catching a PIL exception and raising an OpenSlide one ? I think that's not consistent. It's too much implicit IMHO. I'd prefer to raise the same exception here.